docs/deployment/deploying_custom_image_function.rst

```diff
@@ -183,16 +183,47 @@ Upload and register function
         token=os.environ.get("GATEWAY_TOKEN", "<TOKEN>"),
         host=os.environ.get("GATEWAY_HOST", "<GATEWAY ADDRESS>"),
     )
+    
+    help = """
+    title: custom-image-function
+    description: sample function implemented in a custom image
+    arguments:
+        service: service created with the accunt information
+        circuit: circuit
+        observable: observable 
+    """
 
     function_with_custom_image = QiskitFunction(
         title="custom-image-function",
         image="<image retistory/image name:image tag>",
-        provider="<provider id>"
-    )
+        provider="<provider id>",
+        description=help
```

Comment:
I would improve a little bit this string maybe with a similar format than the previous ones:

```
description="# markdown tex"
```

Reply:
Aaaah ok, I thought it was just the `"help"` string hahaha, got it